lib/platform.test.js

```diff
@@ -170,4 +172,31 @@ describe('Platform', () => {
 			expect(p.hasNcpFirmware).to.be.false;
 		});
 	});
+
+	describe('electron2', () => {
+		it('has electron2 platform definitions by id', () => {
+			const p = platformForId(37);
+			const esomx = platformForName('esomx');
+			expect(p).to.not.be.null;
+			expect(esomx).to.not.be.null;
+
+			expect(p.id).to.equal(37);
+			expect(p.name).to.equal('electron2');
+			expect(p.displayName).to.equal('Electron 2');
+
+			expect(p).excluding(['_id', '_name', '_displayName']).to.deep.equal(esomx);
```

Comment:
Is the comparison to `esomx` just a quick check to make sure we don't modify any of those things by accident in the future?